tools/gn

```diff
@@ -413,8 +413,9 @@ def to_gn_args(args):
       with open(os.path.join(os.path.dirname(__file__), 'fuchsia/target_api_level')) as f:
         gn_args['fuchsia_target_api_level'] = int(f.read().strip())
 
-    # Impeller flags.
-    gn_args['impeller_enable_playground'] = args.enable_impeller_playground
+    if args.target_os == 'mac':
```

Comment:
This isn't quite right. `args.target_os` can be `None` if the target is the host OS, and no OS was specified on the command line. It's probably sufficient to say
```
if args.enable_impeller_playground:
  gn_args['impeller_enable_playground'] = args.enable_impeller_playground
```
If someone sets the flag where it isn't supported, then they'll see the existing warning text.

Reply:
Thanks for reveiw the code, I totally agree with you. 
```
if args.enable_impeller_playground:
  gn_args['impeller_enable_playground'] = args.enable_impeller_playground
```
In this way, the warning log will not appear in unnecessary scenarios, and If someone sets the flag where it isn't supported, then they'll see the existing warning text.